go.mod

```diff
@@ -27,5 +27,6 @@ require (
 	golang.org/x/crypto v0.0.0-20190927123631-a832865fa7ad // indirect
 	golang.org/x/sys v0.0.0-20190927073244-c990c680b611 // indirect
 	golang.org/x/text v0.3.2 // indirect
+	gopkg.in/yaml.v2 v2.2.2
```

Comment:
Can we use `v3` throughout, or do we need to explicitly use `v2`?

Reply:
Good catch!!!! VSCode was excited about v2 - nostalgic, I guess.